kivy_ios/toolchain.py

```diff
@@ -757,6 +759,11 @@ def extract_arch(self, arch):
             ensure_dir(build_dir)
             self.extract_file(self.archive_fn, build_dir)
 
+    @cache_execution
+    def install_hostpython_prerequisites(self):
+        for prerequisite in self.hostpython_prerequisites:
+            _hostpython_pip(["install", prerequisite])
```

Comment:
How about passing the prerequisites list directly?
```suggestion
        _hostpython_pip(["install", self.hostpython_prerequisites])
```
You get the idea

Reply:
The suggestion looked great at a first glance but after getting it a try,  if `self.hostpython_prerequisites == []`(that is the default), the install function crashes badly.

I could opt for a set of conditions, but I feel comfortable with this `for` loop, but let me know what do you think about it.

Reply:
Yeah it was a quick one. We could obviously keep the for loop or go with some fancy things.
```python
self.hostpython_prerequisites and _hostpython_pip(["install", self.hostpython_prerequisites])
```
Or a proper if
```
if self.hostpython_prerequisites:
    _hostpython_pip(["install", self.hostpython_prerequisites])
```
Or even simply check for `args` in `_hostpython_pip()` and return if empty.
But yeah keeping the for loop is completely fine